<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>381</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxEdit">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>251</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>251</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditName"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string> 出生日期</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>  姓  名</string>
        </property>
        <property name="buddy">
         <cstring>name</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditId"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditSex"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditBirth"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>  性 别</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string> 病人编号</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>  病房号</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEditDep"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxChart">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>220</y>
      <width>251</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>查看阶段统计报表</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>查看图表</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxSearch">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>381</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Search</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>261</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,2">
      <property name="horizontalSpacing">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QComboBox" name="comboBox"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditSearch"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>860</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actCheck"/>
   <addaction name="actSave"/>
   <addaction name="actCancel"/>
   <addaction name="actInsert"/>
   <addaction name="actDelete"/>
   <addaction name="actPageUp"/>
   <addaction name="actPre"/>
   <addaction name="actNext"/>
   <addaction name="actPageDown"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actCheck">
   <property name="icon">
    <iconset>
     <normaloff>images/CheckPics(1).jpg</normaloff>images/CheckPics(1).jpg</iconset>
   </property>
   <property name="text">
    <string>查看</string>
   </property>
   <property name="toolTip">
    <string>查看所有数据</string>
   </property>
  </action>
  <action name="actSave">
   <property name="icon">
    <iconset>
     <normaloff>images/save1.bmp</normaloff>images/save1.bmp</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存更改</string>
   </property>
  </action>
  <action name="actCancel">
   <property name="icon">
    <iconset>
     <normaloff>images/ubdo.bmp</normaloff>images/ubdo.bmp</iconset>
   </property>
   <property name="text">
    <string>取消更改</string>
   </property>
   <property name="toolTip">
    <string>取消更改</string>
   </property>
  </action>
  <action name="actInsert">
   <property name="icon">
    <iconset>
     <normaloff>images/316.bmp</normaloff>images/316.bmp</iconset>
   </property>
   <property name="text">
    <string>插入数据</string>
   </property>
   <property name="toolTip">
    <string>插入数据</string>
   </property>
  </action>
  <action name="actDelete">
   <property name="icon">
    <iconset>
     <normaloff>images/318.bmp</normaloff>images/318.bmp</iconset>
   </property>
   <property name="text">
    <string>删除数据</string>
   </property>
   <property name="toolTip">
    <string>删除数据</string>
   </property>
  </action>
  <action name="actNext">
   <property name="icon">
    <iconset>
     <normaloff>images/312.bmp</normaloff>images/312.bmp</iconset>
   </property>
   <property name="text">
    <string>下一条数据</string>
   </property>
   <property name="toolTip">
    <string>下一条数据</string>
   </property>
  </action>
  <action name="actPre">
   <property name="icon">
    <iconset>
     <normaloff>images/310.bmp</normaloff>images/310.bmp</iconset>
   </property>
   <property name="text">
    <string>上一条数据</string>
   </property>
   <property name="toolTip">
    <string>上一条数据</string>
   </property>
  </action>
  <action name="actPageDown">
   <property name="icon">
    <iconset>
     <normaloff>images/314.bmp</normaloff>images/314.bmp</iconset>
   </property>
   <property name="text">
    <string>下一页数据</string>
   </property>
   <property name="toolTip">
    <string>下一页数据</string>
   </property>
  </action>
  <action name="actPageUp">
   <property name="icon">
    <iconset>
     <normaloff>images/308.bmp</normaloff>images/308.bmp</iconset>
   </property>
   <property name="text">
    <string>上一页数据</string>
   </property>
   <property name="toolTip">
    <string>上一页数据</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
